grpo.py

```diff
@@ -392,14 +412,14 @@ def calculate_grpo_objective(
     advantages = advantages.to(device)
     # TODO: check if the advantage is being distributed to padding?
     # Maybe add a mask to the advantages, setting the padding to 0
-    min_product = torch.min(prob_ratios * advantages, clipped_ratios * advantages)
+    min_product = torch.min(prob_ratios * advantages, clipped_ratios * advantages) # TODO: Replace with just prob_ratios * advantages when mu = 1
```

Comment:
This can be a one-liner
```suggestion
    min_product = prob_ratios * advantages if mu ==1 else torch.min(prob_ratios * advantages, clipped_ratios * advantages)
```

Reply:
cool, i will make this change

Reply:
Looks good, thanks for the change.